### PR TITLE
limit tags allowed in bio to prevent faking other fields

### PR DESCRIPTION
**Pull request by LeoMcA**

### CODE REVIEW

File: src/components/profile/ProfileDescription.vue [hunk 1]
```diff
@@ -9,6 +9,8 @@
 import marked from 'marked';
 import insane from 'insane';
 
+insane.defaults.allowedTags = ['a', 'em', 'li', 'ol', 'p', 'strong', 'ul'];
```
Comment (fiji-flo) [inline src/components/profile/ProfileDescription.vue]: Do you know how this translates into markdown?

Comment (LeoMcA) [reply src/components/profile/ProfileDescription.vue]: As in, what markdown we're supporting?

It's just:

`[a](http://example.com)`
`*em*`
```
* <ul>
* <li>
```
```
1. <ol>
2. <li>
```
```
p

p
```
`**strong**`

Comment (fiji-flo) [reply src/components/profile/ProfileDescription.vue]: maybe add `pre` and `code` for \`\`\` and \`

**Comment by LeoMcA** [comment]
@fiji-flo since we now have the border surrounding the bio, I decided to be much more permissive here (while still restricting things like images).

The full list of supported tags can be seen here:

https://dinopark.k8s.test.sso.allizom.org/p/leo

(at least until my profile gets overwritten, so here's the source for posterity:)

    [a](http://example.com)

    > blockquote

    `code`

    ~del~

    *em*

    # h1

    ## h2

    ### h3

    #### h4

    ##### h5

    ###### h6

    1. ol
    1. li

    ```
    pre?
    ```

    <small>small</small>

    **strong**

    <sub>sub</sub>

    <sup>sup</sup>

    * ul
    * li

**Pull request merged**